src/View/Admin/Includes/_navbar.html.twig

```diff
@@ -0,0 +1,15 @@
+<nav class="navbar navbar-expand-sm navbar-dark navAdmin">
+	<div class="container-fluid">
+		<a class="navbar-brand" href="/"><img class="logo" src="/assets/images/logo_min.png" alt="logo Haras">Haras du val de loire</a>
+		<button class="navbar-toggler custom-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+			<span class="navbar-toggler-icon"></span>
+		</button>
+		<div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+			<div class="navbar-nav">
+				<a class="btn-nav-admin" href="/Admin/course">Cours</a>
+				<a class="btn-nav-admin" href="">Stage</a>
+				<a class="btn-nav-admin" href="">Client</a>
```

Comment:
Honnest question : (Using bootstrap for the rest of the navbar but not for the links? Is there a reason?

Reply:
Because bootstrap was not giving me the expected result. The color was difficult to change for so little.